src/wizard/node/profiling-onboarding/node/1.install.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Node
+doc_link: https://discord.gg/zrMjKA4Vnz
+support_level: alpha
+type: language
+---
+
+#### Install
+
+For the Profiling integration to work, you must have the Sentry Node SDK package (minimum version 7.x) installed.
```

Comment:
I believe there's an additional package for the profiling stuff cc @JonasBa

Reply:
Yeah, we need @sentry/tracing and it needs to be imported before @sentry/profiling-node